recipe/meta.yaml

```
@@ -28,16 +26,19 @@ requirements:
     - wheel
   run:
     - python
-    # Extra require
+  run_constrained:
+    - __osx >=10.13
     - pyyaml >=3.10
```

```suggestion
    - pyyaml >=3.10
  run_constrained:
    - __osx >=10.13
```
For consistency with the past version (especially given this is a minor update only) and conda-forge.